config/enabled.yml

```diff
@@ -714,6 +714,12 @@ Style/UnneededPercentQ:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#percent-q'
   Enabled: true
 
+Style/UnneededUnderscore:
```

Comment:
I think we should prepare for making this cop configurable in the future. Some people might say, "Ending a list of variables with a comma is ugly. I want to enforce the use of underscores.". So the name should not reflect any opinion about whether to use the underscores or not.

Some suggestions:
* `AnonymousUnusedVariable`
* `ThrowawayVariable`
* `UnderscroreVariable`